### PR TITLE
fixed xsoar linter

### PR DESCRIPTION
**Pull request by orhovy**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/content/pull/8813
https://github.com/demisto/demisto-sdk/pull/718

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by orhovy** [comment]
@bakatzir  validate would not pass as I didnt update the DBotPredictPhishingWords docker image from 1.3 to languages1.0 as requested in the previous pr.

**Review by bakatzir — APPROVED** [review]

**Comment by bakatzir** [comment]
@amshamah419 Can you please help us with the RN?

File: Packs/Base/ReleaseNotes/1_2_2.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### DBotPredictPhishingWords
+- Removed the usage of sys-exit.
```
Comment (amshamah419) [inline Packs/Base/ReleaseNotes/1_2_2.md:4]: Change to the maintenance and stability fixes.

**Comment by orhovy** [comment]
@bakatzir build failures does not seem to be related to this prs changes. and validate fails as a result of an old docker image and it is intended to be that way.

**Comment by bakatzir** [comment]
All on a limit error from Json WHOIS. merging.

**Pull request merged**